backend/kernelCI_app/views/testsByTreeAndCommitHashView.py

```diff
@@ -40,7 +40,7 @@ def get(self, request, commit_hash: str | None):
         query = Tests.objects.raw(
             f"""
                 SELECT t.id, c.git_repository_url,
-                c.git_commit_hash, t.id, t.build_id, t.start_time,
+                c.git_commit_hash, t.id, t.build_id, t.start_time, t.duration,
```

Comment:
since you are touching this endpoint, can you remove the f-string here? 
you only need to replace with parametrized query like here https://github.com/kernelci/dashboard/pull/178/files#diff-c4866cf8cc40c84d094af4cc71e7b18a4d0973c4d05ab4707ac66696439aac6e